includes/gateway/class-omise-payment.php

```diff
@@ -74,16 +74,38 @@ abstract class Omise_Payment extends WC_Payment_Gateway {
 	 */
 	protected $order;
 
+	protected $enabled_processing_notification = false;
+
 	public function __construct() {
 		$this->omise_settings   = Omise()->settings();
 		$this->payment_settings = $this->omise_settings->get_settings();
 
 		add_action( 'wp_enqueue_scripts', array( $this, 'omise_checkout_assets' ) );
+		add_action( 'woocommerce_order_status_processing', array($this, 'email_processing_admin_notification'), 10, 2 );
 		add_filter( 'woocommerce_email_recipient_new_order', array($this, 'disable_merchant_order_on_hold_email'), 10, 2 );
     }
 
+	/**
+	 * Due to payment method like paynow the new order email was disable then customize plugin to send when processing instead
+	 * but need to resend for other payment method otherwise merchant will get duplicate email once new order created
+	 * @param string $order_id
+	 * @param string|WC_Order $order
+	 */
+	public function email_processing_admin_notification( $order_id, $order ) {
+		$payment_gateway = wc_get_payment_gateway_by_order( $order );
+		if (is_a( $payment_gateway, 'Omise_Payment' ) && $payment_gateway->enabled_processing_notification) {
+			WC()->mailer()->get_emails()['WC_Email_New_Order']->trigger( $order_id );
+		}
+	}
```

Comment:
not sure if it's worth refactoring this into a helper or service class. Benefit is we can move this logic out of this class and we can write some unit tests and mock a few orders to see they work as expected.

Reply:
Let's me see